### PR TITLE
fix envp

### PR DESCRIPTION
**Pull request by folkertdev**

### CODE REVIEW

File: crates/cli/src/lib.rs [hunk 1]
```diff
@@ -814,12 +814,17 @@ fn make_argv_envp<'a, I: IntoIterator<Item = S>, S: AsRef<OsStr>>(
     // envp is an array of pointers to strings, conventionally of the
     // form key=value, which are passed as the environment of the new
     // program.  The envp array must be terminated by a NULL pointer.
```
Comment (folkertdev) [inline crates/cli/src/lib.rs:816]: I wrote this comment, which is correct, but then the implementation was not, actually. Fun.

**Comment by folkertdev** [comment]
open question: how do we test this?

**Comment by folkertdev** [comment]
Looks good, want to merge it @ayazhafiz

**Review by ayazhafiz — APPROVED** [review]
Nice work Folkert

**Pull request merged**